targets/targets.json

```diff
@@ -144,7 +144,7 @@
         ],
         "extra_labels": [
             "TFM",
-            "TFM_V1_1",
+            "TFM_LATEST",
```

Comment:
I agree that using a generic name like `TFM_LATEST` simplifies the maintenance but it is less explicit than before. Since you explained in your documentation PR how to map `TFM_LATEST` to the actual TFM release that is supported, this should be fine.